package.json

```diff
@@ -157,6 +157,7 @@
     "tree-kill": "^1.0.0",
     "ts-node": "^1.3.0",
     "tslint": "^3.11.0",
-    "walk-sync": "^0.2.6"
+    "walk-sync": "^0.2.6",
+    "webpack": "^2.1.0-beta.23"
```

Comment:
A new project may look like this. 

![screen shot 2016-09-20 at 10 36 30 am](https://cloud.githubusercontent.com/assets/3408176/18677346/1cb63d10-7f1e-11e6-86cf-cd00c4f892bc.png)